github/issues.go

```diff
@@ -36,6 +36,7 @@ type Issue struct {
 	Milestone        *Milestone        `json:"milestone,omitempty"`
 	PullRequestLinks *PullRequestLinks `json:"pull_request,omitempty"`
 	Repository       *Repository       `json:"repository,omitempty"`
+	Reactions        *ReactionsSummary `json:"reactions,omitempty"`
```

Comment:
Is there a reason the type `ReactionsSummary` isn't named `Reactions`? That would be more consistent with the json field name (here and elsewhere). It'd also be more consistent with the other similar types (see lines above).